packages/esm-active-visits-app/src/pagination/pagination.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useMemo } from 'react';
+import Pagination from 'carbon-components-react/es/components/Pagination';
+import styles from './pagination.component.scss';
+import { useConfig } from '@openmrs/esm-framework';
+
+interface ActiveVisitsPaginationProps {
+  currentItems: number;
+  totalItems: number;
+  pageNumber: number;
+  pageSize: number;
+  onPageNumberChange?: any;
+  pageUrl?: string;
+}
+
+const ActiveVisitsPagination: React.FC<ActiveVisitsPaginationProps> = ({
+  totalItems,
+  pageSize,
+  onPageNumberChange,
+  pageNumber,
+}) => {
+  const config = useConfig();
+
+  const pageSizes = useMemo(() => {
```

Comment:
@FlorianRappl I am talking about the select option as shown in the video(uploading) below. It's being used to make the select option

Reply:
I know. But again - read my comment. No need for any computations.

Reply:
It's again a case of making a video / comment / whatever instead of carefully reading what I wrote. If unclear, please message me on Slack!